pylint_django/tests/input/external_django_tables2_noerror_meta_class.py

```diff
@@ -2,7 +2,7 @@
 # don't produce old-style-class warnings, see
 # https://github.com/PyCQA/pylint-django/issues/56
 
-# pylint: disable=missing-docstring,too-few-public-methods
+# pylint: disable=model-no-explicit-str,model-no-explicit-str,missing-docstring,too-few-public-methods
```

Comment:
Some of the files have this repeated twice! Look below.  This could also be the cause for flake8 complaining.

Reply:
Ah I thought I caught these already, my `sed` to insert these went wrong a few times and I thought I backed out the changes but seems I have missed a couple.